iOS/baseball-game/baseball-game/AppCycle/SceneDelegate.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  SceneDelegate.swift
+//  baseball-game
+//
+//  Created by Song on 2021/05/03.
+//
+
+import UIKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+
+    var window: UIWindow?
+    
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        
+        guard let windowScene = (scene as? UIWindowScene) else { return }
+        window = UIWindow(windowScene: windowScene)
+
+        let rootViewController = UIStoryboard(name: "Main", bundle: nil).instantiateViewController(identifier: "LoginViewController") as LoginViewController
+
+        let navigationViewController = UINavigationController(rootViewController: rootViewController)
+
+        window?.rootViewController = navigationViewController
+    }
```

Comment:
> 코드 형식 

파일들을 살펴보니 class, extension, struct 등등의 맨 처음과 맨 끝은 띄우고, 함수는 한 칸씩 공백을 띄우는 코드 형식인 것 같네요!
```suggestion
    }
    
```
=> 제 말이 맞다면 위처럼 바뀌어야 될 것 같습니다~